management/README.md

```diff
@@ -4,14 +4,136 @@ This directory contains the configuration needed to setup a management GKE clust
+    kpt cfg set ./instance name $NAME
+    kpt cfg set ./instance location $LOCATION
+    kpt cfg set ./instance gcloud.core.project $PROJECT
+    ```
+    * NAME is the cluster name of your management cluster. Management cluster
+    should be a different cluster from your Kubeflow cluster.
+
+        Note, NAME should
+        * start with a lowercase character
+        * only contain lowercase characters, numbers and `-`
+        * end with a number or character
+        * contain no more than 18 characters
+    * LOCATION is the management cluster's location, you can choose between regional or zonal.
+    * PROJECT is the Google Cloud project where you will create this management cluster.
+
+    It stores values you set in `./instance/Kptfile`. Commit the
+    changes to source control to preserve your configuration.
+
+    You can learn more about `kpt cfg set` in [kpt documentation](https://googlecontainertools.github.io/kpt/reference/cfg/set/).
+
+1. Create/apply the management cluster:
+    ```bash
+    make apply
+    ```
+1. Create a kubeconfig context for the cluster:
+    ```bash
+    make create-ctxt
+    ```
+    The context will be called `${NAME}`.
+1. Install the config connector:
+    ```bash
+    make apply-kcc
+    ```
+
+## Clean up Instructions
+The following instructions introduce how to clean up all resources created when
+installing management cluster and using management cluster to manage Google
+Cloud resources in the managed project.
+
+Be cautious, if you want to keep managed Google Cloud resources, you should
+finish steps below to keep managed Google Cloud resources before cleaning up
+management cluster.
+
+### Clean up or keep managed Google Cloud resources
+There are Google Cloud resources managed by Config Connector in the
+management cluster after you deploy Kubeflow clusters with this management
+cluster.
+
+To delete all the managed Google Cloud resources, delete the managed project
+namespace:
+```bash
+kubectl use-context ${MGMTCTXT}
+kubectl delete namespace --wait ${MANAGED_PROJECT}
+```
+To keep these Google Cloud resources after deleting cnrm resources, you should
+add annotation `cnrm.cloud.google.com/deletion-policy: abandon` to all of them
+**before** deleting the namespace.
+
+Refer to
+[Config Connector: Keeping resources after deletion](https://cloud.google.com/config-connector/docs/how-to/managing-deleting-resources#keeping_resources_after_deletion)
+for more details.
+
+### Clean up management cluster Google Cloud resources
+
+To delete the IAM policy binding for the managed project, the Google service account,
+and the management cluster:
+```bash
+gcloud projects remove-iam-policy-binding ${MANAGED_PROJECT} \
+    --member=serviceAccount:${NAME}-cnrm-system@${PROJECT}.iam.gserviceaccount.com \
+    --role=roles/owner
+gcloud --project=${PROJECT} iam service-accounts delete \
+    ${NAME}-cnrm-system@${PROJECT}.iam.gserviceaccount.com
+gcloud --project=${PROJECT} container clusters delete \
```

Comment:
[ASK]
Once a management cluster is used to deploy kubeflow.
If we remove the associated resources and delete the management cluster. Will a user be able to upgrade or do operation on the same deployed kubeflow in future? by creating a new management cluster with the same configurations?

In my case, we keep the management cluster at size zero as it is not in use.

Reply:
Yes, that's possible.
https://cloud.google.com/config-connector/docs/how-to/managing-deleting-resources#acquiring_a_dataset

Reply:
Great! thank you!